packages/mjml-text/src/index.js

```diff
@@ -65,6 +65,7 @@ class Text extends Component {
         fontStyle: mjAttribute('font-style'),
         fontWeight: mjAttribute('font-weight'),
         lineHeight: mjAttribute('line-height'),
+        letterSpacing: mjAttribute('letter-spacing', "px"),
```

Comment:
Hey ! `mjAttribute` has no 2nd params, you should use the `defaultUnit` function if you want so